snuba/datasets/processors/metrics_aggregate_processor.py

```diff
@@ -83,6 +83,7 @@ def process_message(
                 "org_id": _literal(message["org_id"]),
                 "project_id": _literal(message["project_id"]),
                 "metric_id": _literal(message["metric_id"]),
+                "use_case_id": _literal(message.get("use_case_id", "sessions")),
```

Comment:
Just for documentation purposes -- I don't think we actually use the `AggregateProcessor` anywhere. But it's good to add for safety. We'll likely deprecate `AggregateProcessor` soon